lib/hydra/works/models/concerns/generic_file/contained_files.rb

```diff
@@ -10,6 +10,9 @@ module Hydra::Works::GenericFile::ContainedFiles
     directly_contains_one :original_file, through: :files, type: ::RDF::URI("http://pcdm.org/use#OriginalFile"), class_name: "Hydra::PCDM::File"
     directly_contains_one :thumbnail, through: :files, type: ::RDF::URI("http://pcdm.org/use#ThumbnailImage"), class_name: "Hydra::PCDM::File"
     directly_contains_one :extracted_text, through: :files, type: ::RDF::URI("http://pcdm.org/use#ExtractedText"), class_name: "Hydra::PCDM::File"
+    directly_contains_one :mp3, through: :files, type: ::RDF::URI("http://pcdm.org/use#ServiceFile"), class_name: "Hydra::PCDM::File"
```

Comment:
I believe Sufia only includes these 4 if you've explicitly turned on audiovisual processing (which is off by default). I'm in favor of striking these four lines and creating a ticket in Sufia::Core that says "Add these back here at this level because they're not a core Hydra::Works concern" or some such.  Reasonable?